packages/apps/docs/src/pages/build/guides/building-a-voting-dapp.md

```diff
@@ -244,8 +244,8 @@ Don't forget to add the snippet above in the `election` module body.
 :::info
 
 To learn more about guards and capabilities, please visit the
-[Guards, Capabilities and Events](/pact/reference/concepts#guards) section of
-the Pact official documentation.
+[Guards, Capabilities and Events](/pact/reference/concepts#guardsh2143724174)
+section of the Pact official documentation.
 
 :::
 
```

Comment:
this file needs to be removed and have to update the new election-dapp sections here.